src/components/Search/Search.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import './Search.modules.scss';
+
+export const Search = () => {
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(e.target[1].value);
+  };
+
+  return (
+    <section className='search__content'>
+      <h1>FIND YOUR MOViE</h1>
+      <form onSubmit={handleSubmit}>
+        <label htmlFor='search'>
+          <input
```

Comment:
You have installed `antd` library so u feel free to use as much components as possible from it, for example `Input`:

https://ant.design/components/input/